backend/seeds/seed.py

```diff
@@ -0,0 +1,14 @@
+import json
+from database.database import get_db
+from symptoms.models import Symptom
+
+# Using `next` here because get_db() returns a generator
+session = next(get_db())
+
+with open('data.json', 'r') as json_data:
```

Comment:
One request, and plenty of suggestions, depending on your opinion on them, how much time you want to invest here, and how long you thing they might take:

* **The request:** Let's make `'data.json'` a variable defined at the top of the file. Something like `SEED_FILE='data.json'` would do. This is only intended to make more specific the place where one would tweak the code if we want to change the input file, and easier to find without reading the whole thing.
* **The suggestions:**
    - Maybe this could all be within the `database` app? sounds reasonable idk
    - It'd be cool to have a description of what the script does and its intention at the top of the file. I'm thinking something like [what we did in dearduck](https://github.com/dearduck/dearduck/blob/master/backend/scripts/product_scrapping/import_products.py), only way simpler because the script is simpler and frankly Martin went a bit overboard on those.
    - It would also be super cool to have instructions to use this in the README!
    - While this approach is a-ok, for some models we might want to input a LOT of data (think, COVID cases per state in the US idk) and they would make the original JSON a bit hard to read. Basing this off a list of files instead of a single one might be something to consider. But maybe I'm overengineering 😅

Reply:
I'm taking the requests and all the suggestions but the last one. Let's do that when time comes.
I can do them in half and hour or so, and they all worth it

Reply:
I ended up applying all the suggestions. All of them were super straightforward 😄